CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+#### Contributing to a Manim Community Fork - General guidelines
+
+As this is a Fork of [Manim from 3b1b](https://github.com/3b1b/manim), contributing can be a bit confusing. So here is a small guide on it. 
+
+1. First, make a fork of either this Repo or the Repo from which this was Forked. ( 3b1b Repository)
+2. Then clone the repository you forked by the command below.
+
+ ```sh
+   git clone <your-fork-url>
+   cd manim
+ ```
+
+3. Now that you have cloned the Repo, before making any changes, you have to enter the commands below to contribute to Manim Community Fork.
+
+   ```sh
+   git remote add fork https://github.com/ManimCommunity/manim.git
+   git fetch fork
+   git checkout -b <your-branch-name> fork/master
+   ```
+
+   The first command says `git` that you are going to make a Pull Request for Manim Community. 
+   Then the second command pulls all the commits from the Fork ( Community Fork ).
+   The third one changes your current working branch even with Manim Community Fork master.
+
+4. After that, you can make your changes to contents inside the `manim` directory. Then, you can commit the changes.
+
+5. Finally, before typing in `git push` enter the command below.
+
+   ```sh
+   git push -u origin <your-branch-name>
+   ```
+
+   Doing this creates a new branch with the updated contents in Github. 
```

Comment:
"Doing this creates a new branch with the updated contents in your fork on GitHub.